resource_mock/resources.py

```diff
@@ -41,11 +46,17 @@ def is_instrument_available(self, site: Site, instrument: str) -> bool:
     def is_disperser_available(self, site: Site, disperser: str) -> bool:
         return disperser in self.gratings[site]
 
-    def is_mask_available(self, site: Site, fpu_mask: str) -> bool:
+    def is_mask_available(self,site: Site, fpu_mask: str) -> bool:
+
         barcode = None
+        if fpu_mask == 'None':
+            return True
         if fpu_mask in self.fpu_to_barcode[site]:
             barcode = self.fpu_to_barcode[site][fpu_mask]
         elif '-' in fpu_mask:
-            barcode = Resources._decode_mask(fpu_mask)
-        return barcode and barcode in self.fpur[site]
-
+            barcode = self._decode_mask(fpu_mask)
```

Comment:
I'm not sure if some of these changes back came about because of problems with rebasing, but since `_decode_mask` can be written as a static method, we should be accessing it with `Resources._decode_mask` instead of `self._decode_mask`.